libs/openchallenges/api-description/src/paths/challengePlatforms/@{challengePlatformName}.yaml

```diff
@@ -4,8 +4,8 @@ get:
   tags:
     - ChallengePlatform
   summary: Get a challenge platform
-  description: Returns the challenge platform specified
-  operationId: getChallengePlatform
+  description: Returns the challenge platform specified by its unique name
+  operationId: getChallengePlatformByName
```

Comment:
@vpchung This is the unconventional endpoint that retrieve a resource based on its name and not its Id or login. I think that we are not using this endpoint in the web app. Could you please check? If we don't use it, then we should remove it.